src/app/public/modules/country-field/country-field.component.html

```diff
@@ -0,0 +1,59 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <span
+    class="sky-country-field-container"
+    [ngClass]="{
+      'sky-country-field-disabled': disabled
+    }"
+  >
+    <form
+      novalidate
+      class="sky-country-field-country-search"
+      [formGroup]="countrySearchForm"
+    >
+      <sky-autocomplete
+        [data]="countries"
+        [propertiesToSearch]="['name', 'iso2']"
+        [searchResultTemplate]="countrySearchResultTemplate"
+        [searchTextMinimumCharacters]="2"
+        (selectionChange)="onCountrySelected($event)"
+      >
+        <div
+          class="sky-form-control sky-country-field-input"
+          [ngClass]="{
+            'sky-field-status-active': isInputFocused
+          }"
+        >
+          <div *ngIf="selectedCountry && selectedCountry.iso2 && !isInPhoneField"
+            [className]="'sky-country-field-flag iti-flag ' + selectedCountry.iso2.toLowerCase()"
+          ></div>
+          <textarea
+            skyAutocomplete
+            formControlName="countrySearch"
+            name="selectedCountry"
+            [attr.aria-label]="'skyux_country_field_search_placeholder' | skyLibResources"
+            [attr.disabled]="disabled ? true : undefined"
+            [placeholder]="'skyux_country_field_search_placeholder' | skyLibResources" #countrySearchInput
+          ></textarea>
+        </div>
+      </sky-autocomplete>
+    </form>
+  </span>
+</div>
+
+<ng-template
+  let-item="item"
+  #countrySearchResultTemplate
+>
+  <div
+    [className]="'iti-flag ' + item.iso2"
```

Comment:
Does this also need `.toLowerCase()` like the other line above?

Reply:
In theory no because it will go off the list of countries that the library gives and not potential user input. But better safe than sorry. I've added that :)